physics/progsigma_calc.f90

```diff
@@ -217,12 +217,14 @@ subroutine progsigma_calc (im,km,flag_init,flag_restart,           &
          do i= 1, im
             if(cnvflg(i)) then
                sigmab(i)=sigmab(i)/betascu
+               sigmab(i)=MAX(0.01,sigmab(i))
```

Comment:
This is a coding style choice, but IMO, the new line here and at 227 can be combined with the previous lines since they're just putting a lower bound on the calculation of `sigmab`. Not a big deal though.